pkgs/development/python-modules/types-freezegun/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "types-freezegun";
+  version = "1.1.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0kxiv0yjbbvp1zx694ir149b26kjzvb6600fh397v32b8jvs8w2w";
+  };
+
+  # Module doesn't have tests
+  doCheck = false;
```

Comment:
It looks like unit tests are available on github: https://github.com/python/typeshed/tree/master/tests. Otherwise, you could use [`pythonImportsCheck`](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck).

Reply:
In the past were those tests not for the stubs itself. I guess that I should have another look.

Yes, `pythonImportsCheck` was missing. Is addressed by #155528.